### ISSUE TITLE
Encoding problem in Windows

### ISSUE THREAD

**Issue by tobegit3hub** [mkdocs/mkdocs#239]
When I run `mkdocs` in Windows, it fails and here's the log.

```
/D/Projects
$ mkdocs new my-p
Creating project directory: my-p
Writing config file: my-p\mkdocs.yml
Writing initial docs: my-p\docs\index.md
Traceback (most recent call last):
  File "c:\Python34\Scripts\mkdocs-script.py", line 9, in <module>
    load_entry_point('mkdocs==0.11.1', 'console_scripts', 'mkdocs')()
  File "C:\Python34\lib\site-packages\mkdocs\main.py", line 60, in run_main
    main(cmd, args=sys.argv[2:], options=dict(opts))
  File "C:\Python34\lib\site-packages\mkdocs\main.py", line 45, in main
    new(args, options)
  File "C:\Python34\lib\site-packages\mkdocs\new.py", line 51, in new
    open(index_path, 'w').write(index_text)
UnicodeEncodeError: 'gbk' codec can't encode character '\xa0' in position 348: i
llegal multibyte sequence
```

It seems to the problem about encoding. I think it would be better to support because python itself is cross-platform.

**Comment by vbd** [comment]
Desktop Windows 8 + Python 2.7.9: similar error message
Notebook Windows 8.1 + Python 3.4.3: no error message, works as expected.

**Comment by waylan** [comment]
This is almost certainly a problem with using the built-in `open` function without defining the encoding. In my experience, for cross-platform consistency, you must define the encoding when you open the file, not later. The `codecs` lib certainly addresses this, however, the `io` lib was added in Python 2.6 and it provides the Python 3 implementation of `open` in [Python 2][1]. In fact, the Python 3 [docs][3] state that `io.open` is "an alias for the built-in open() function."

So, I expect the fix for this is to import `open` from `io` and then do:

```python 
input_content = open(input_path, 'r', encoding='utf-8').read()
```

I'll test this on my windows box tonight.

[1]: https://docs.python.org/2/library/io.html#io.open
[2]: https://docs.python.org/3.4/library/io.html#io.open

**Comment by waylan** [comment]
Hmm, I can't seem to reproduce this on my Windows 7 box. That said, I still think the above change should be made.

**Comment by waylan** [comment]
Those of you experiencing this problem, can you verify whether #428 fixes the issue for you?

**Comment by d0ugal** [comment]
This should be fixed by #428 - happy to reopen if somebody can confirm this is still an issue.
